packages/elements-core/src/components/TryIt/TryIt.tsx

```diff
@@ -197,81 +214,94 @@ export const TryIt: React.FC<TryItProps> = ({
+          examples={mediaTypeContent.examples ?? []}
+          requestBody={textRequestBody}
+          onChange={setTextRequestBody}
+        />
+      ) : null}
+
+      <Panel.Content className="SendButtonHolder" mt={4} pt={!isOnlySendButton && !embeddedInMd ? 0 : undefined}>
+        <HStack alignItems="center" spacing={2}>
+          <Button appearance="primary" loading={loading} disabled={loading} onPress={handleSendRequest} size="sm">
+            Send API Request
+          </Button>
 
-        {formDataState.isFormDataBody ? (
-          <FormDataBody
-            specification={formDataState.bodySpecification}
-            values={bodyParameterValues}
-            onChangeValues={setBodyParameterValues}
-          />
-        ) : mediaTypeContent ? (
-          <RequestBody
-            examples={mediaTypeContent.examples ?? []}
-            requestBody={textRequestBody}
-            onChange={setTextRequestBody}
-          />
-        ) : null}
-
-        <Panel.Content className="SendButtonHolder">
-          <Flex alignItems="center">
-            <Button appearance="primary" loading={loading} disabled={loading} onPress={handleClick} size="sm">
-              Send Request
-            </Button>
-
-            {mockUrl && (
-              <MockingButton options={mockingOptions} onOptionsChange={setMockingOptions} operation={httpOperation} />
-            )}
-          </Flex>
-
-          {validateParameters && hasRequiredButEmptyParameters && (
-            <Box mt={4} color="danger-light" fontSize="sm">
-              <Icon icon={faExclamationTriangle} className="sl-mr-1" />
-              You didn't provide all of the required parameters!
-            </Box>
+          {servers.length > 1 && <ServersDropdown servers={servers} />}
+
+          {isMockingEnabled && (
+            <MockingButton options={mockingOptions} onOptionsChange={setMockingOptions} operation={httpOperation} />
           )}
-        </Panel.Content>
+        </HStack>
+
+        {validateParameters && hasRequiredButEmptyParameters && (
+          <Box mt={4} color="danger-light" fontSize="sm">
+            <Icon icon={faExclamationTriangle} className="sl-mr-1" />
+            You didn't provide all of the required parameters!
+          </Box>
+        )}
+      </Panel.Content>
+    </>
+  );
+
+  let tryItPanelElem;
+
+  // when TryIt is embedded, we need to show extra context at the top about the method + path
+  if (embeddedInMd) {
+    tryItPanelElem = (
+      <Panel isCollapsible={false} p={0} className="TryItPanel">
+        <Panel.Titlebar bg="canvas-300">
+          <Box fontWeight="bold" color={!isDark ? HttpMethodColors[httpOperation.method] : undefined}>
+            {httpOperation.method.toUpperCase()}
+          </Box>
+          <Box fontWeight="medium" ml={2} textOverflow="truncate" overflowX="hidden">
+            {`${chosenServer?.url || ''}${httpOperation.path}`}
+          </Box>
+        </Panel.Titlebar>
+
+        {tryItPanelContents}
```

Comment:
Hmm, since we are wrapping the content here anyways, I wonder how feasible it would be to literally define a `EmbeddableTryIt` component, that would simply use `TryIt`.

Would reduce the ifology and make `TryIt` itself more streamlined? (TryIt is already borderline out of control complex IMHO and I don't want to aggravate the situation).

Reply:
Yeah, I had to reeeallly resist refactoring the entire TryIt component 😊. How about I re-arrange to pull relevant bits out into a `TryItRequest` component (complement to the `TryItResponse` we already have), you good with that?

Reply:
Eh - it's a fairly large change (I still think would be good to do) and we'd have to re-think the state since `RequestSamples` needs access to the state from `TryItRequest`. Prob best in separate PR.